classes/Hook/HookDisplayFooter.php

```diff
@@ -49,7 +50,8 @@ public function __construct(Ps_Googleanalytics $module, Context $context)
      */
     public function run()
     {
-        $gaTools = new GoogleAnalyticsTools();
+        $isV4Enabled = (bool) Configuration::get('GA_V4_ENABLED');
```

Comment:
assign `$currencyIsoCode = $this->context->currency->iso_code;` at top of function,
then use `$currencyIsoCode` in the rest looks better

Reply:
Fixed